sktime/forecasting/online_learning/_online_ensemble.py

```diff
@@ -47,15 +52,12 @@ def fit(self, y, X=None, fh=None):
         self : returns an instance of self.
         """
 
-        self._set_y_X(y, X)
-        self._set_fh(fh)
         names, forecasters = self._check_forecasters()
         self.weights = np.ones(len(forecasters)) / len(forecasters)
         self._fit_forecasters(forecasters, y, X, fh)
-        self._is_fitted = True
         return self
 
-    def _fit_ensemble(self, y, X=None):
+    def fit_ensemble(self, y, X=None):
```

Comment:
This should still be private, no?
```suggestion
    def _fit_ensemble(self, y, X=None):
```

Reply:
yes, why was this changed?

Reply:
Must have been an error